lib/commands/endTest.js

```diff
@@ -16,15 +16,16 @@ const logger = require('../utils/logger');
  * @returns {ChainablePromise.<void>}
  */
 async function endTest() {
-	logger.info(commandWillBeExecuted('endTest'));
+	const info = logger.delayed(commandWillBeExecuted('endTest'));
 	// authorize
 	const authedContent = await authContext.authorizeWs({type: wsContentTypes.endTest}, endTest.name);
 
 	// make ws request
 	await webSockets.send(authedContent);
 
 	testContext.clear();
-	logger.info(commandExecuted('endTest'));
+	clearInterval(info);
```

Comment:
they are not actually immediate. `logger.delayed()` shows the log line only if there is a longer period of log silence, so that user is comforted that something is happening but it takes time. If another log would come before the timeout is reached the delayed log line should not be displayed anymore. 

However you are right about the weird `clearTimeout` which should have been an implementation detail of the logger, i'll fix that